powershell/resources/psruntime/BuildTime/Cmdlets/ExportProxyCmdlet.cs

```diff
@@ -42,7 +42,7 @@ public class ExportProxyCmdlet : PSCmdlet
         [ValidateNotNullOrEmpty]
         public string DocsFolder { get; set; }
 
-        [Parameter(Mandatory = true, ParameterSetName = "Docs")]
+        [Parameter(Mandatory = true)]
```

Comment:
I don't think you need to alter the ParameterSetName since examples are part of docs. Consider adding an ` if (!ExcludeDocs)` check at this point instead https://github.com/Azure/autorest.powershell/blob/6390998136582908155e819cc9393aa95bd2c33a/powershell/resources/psruntime/BuildTime/Cmdlets/ExportProxyCmdlet.cs#L76-L83

Reply:
The change is correct. Examples folder should be provided in both cases is because we need these examples for comment-based help. We will put these examples from the examples folder in the header of exports/*.ps1